docs/resources/maintenance_window.md

```
@@ -10,7 +9,103 @@
+
+## Time Format
+
+Time should be specified in `HH:MM` format (24-hour format):
+- `09:00` - 9:00 AM
+- `23:30` - 11:30 PM
+- `00:00` - Midnight
+
+## Weekdays
+
+For weekly maintenance windows, specify days as numbers:
+- `1` - Monday
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)